portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,55 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import java.lang.String;
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+// @WebServlet("/name")
+// public class DataServlet extends HttpServlet {
+
+// ArrayList<String> numList = new ArrayList<>();
+
+//   @Override
+//   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+//     numList.add("1");
+//     numList.add("2");
+//     numList.add("3");
+//     response.setContentType("text/html;");
+//     response.getWriter().println(convertToJsonUsingGson(numList));
+//   }
+
+//   private String convertToJsonUsingGson(ArrayList<String> numList) {
+//     Gson gson = new Gson();
+//     String json = gson.toJson(numList);
+//     return json;
+//   }
+
+
+// }
+
+@WebServlet("/receive-email")
```

Comment:
If you do this, the rest of the instructions will be more confusing, since you don't have a DataServlet at "/data" anymore. Why not leave this as "/data" and use "/data" as your action in your HTML form?